R/recipe.R

```diff
@@ -779,7 +779,13 @@ utils::globalVariables(c("number"))
 
 # ------------------------------------------------------------------------------
 
-#' @rdname required_pkgs
+#' S3 methods for tracking which additional packages are needed for steps.
+#'
+#' @param x A recipe or recipe step
+#' @param infra Should recipes itself be included in the result?
+#' @return A character vector
+#' @name required_pkgs.recipe
+#' @keywords internal
 #' @export
 required_pkgs.recipe <- function(x, infra = TRUE, ...) {
```

Comment:
I went with naming the rd file `required_pkgs.recipe` rather than `required_pkgs.step` because I document it with the recipe method and I figured that made more sense